week-4/Homework/mandatory/4-groceries.js

```diff
@@ -28,10 +28,11 @@ Exercise 1:
   Then use console.log() to print out the list.
 */
 // Gather all week item names into this array
-let weeklyGroceriesToBuy = [];
-
 
+let weeklyGroceriesToBuy = [];
 
+weeklyGroceriesToBuy = Object.values(weeklyMealPlan);
```

Comment:
this results in an array of arrays
we want one array with all the items,
you will need to do some looping